base/metrics/field_trial_unittest.cc

```diff
@@ -923,6 +923,8 @@ class FieldTrialListTest : public ::testing::Test {
   test::ScopedFeatureList scoped_feature_list_;
 };
 
+// TODO(b/298237462): Try to enable |FieldTrialList|.
```

Comment:
FYI, here is the bug b/316198056 for recording which test we want to enable later.